packages/datatrak-web/src/views/AccountSettingsPage/RequestCountryAccessSection/RequestableCountryChecklist.tsx

```diff
@@ -0,0 +1,96 @@
+/*
+ * Tupaia
+ * Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ */
+
+import React from 'react';
+import styled from 'styled-components';
+import { useFormContext } from 'react-hook-form';
+import { Checkbox } from '@tupaia/ui-components';
+import { Entity, Project, TupaiaWebCountryAccessListRequest } from '@tupaia/types';
+import { theme } from '../../../theme';
+
+const Container = styled.fieldset`
+  margin: 0;
+  padding-block: 0;
+
+  border-radius: 0.1875rem;
+  border: 1px solid ${theme.palette.grey[400]};
+  block-size: 100%;
+  overflow-y: auto; /* fallback */
+  overflow-block: auto;
+  padding-inline: 0.87rem;
+
+  // Match styling of ui-components TextField
+  :disabled {
+    color: ${theme.palette.text.secondary};
+    background-color: ${theme.palette.grey['100']};
+  }
+`;
+
+const StyledCheckbox = styled(Checkbox).attrs({ color: 'primary' })`
+  margin-block: 0;
+
+  .MuiFormControlLabel-root {
+    inline-size: 100%;
+  }
+
+  .MuiFormControlLabel-label {
+    font-size: 0.875rem;
+    inline-size: 100%;
+    line-height: 1.125rem;
+  }
+`;
+
+interface RequestableCountryChecklistProps {
+  projectCode: Project['code'];
+  countries: TupaiaWebCountryAccessListRequest.ResBody;
+  selectedCountries: Entity['id'][];
+  setSelectedCountries: React.Dispatch<React.SetStateAction<Entity['id'][]>>;
+  disabled?: boolean;
+}
+
+export const RequestableCountryChecklist = ({
+  projectCode,
+  countries,
+  selectedCountries,
+  setSelectedCountries,
+  disabled,
+}: RequestableCountryChecklistProps) => {
+  const { register } = useFormContext();
+
+  const selectCountry = (id: Entity['id'], select = true) =>
+    setSelectedCountries(
+      select ? [...selectedCountries, id] : selectedCountries.filter(element => element !== id),
+    );
+
+  function getTooltip(hasAccess: boolean, hasRequestedAccess: boolean) {
```

Comment:
Just should standardise whether we use `function` or `const` I think, as opposed to having both